src/boosteriferous/boosteriferous.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What changed here?  Did a BOM appear, because those are evil and don't belong in UTF-8 grr.  YASW (yet another stupid Windowsism).